homeassistant/components/lupusec/entity.py

```diff
@@ -0,0 +1,48 @@
+"""Provides the Lupusec entity for Home Assistant."""
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class LupusecDevice(Entity):
+    """Representation of a Lupusec device."""
+
+    _attr_has_entity_name = True
+
+    def __init__(self, data, device, config_entry) -> None:
+        """Initialize a sensor for Lupusec device."""
+        self._data = data
+        self._device = device
+        self._entry_id = config_entry.entry_id
+        self._attr_unique_id = self.get_unique_id(
+            config_entry.entry_id, device.device_id
+        )
```

Comment:
I think it's better to set this a bit different. Unique id should be unique for entity platform + domain combination. So setting it to `device_id` would work just fine. In the alarm_control_panel you could overwrite it to set the entry_id instead. (This way you have the logic about the unique_id and stuff located at the right file)

Reply:
Done.